src/services/donationService.ts

```diff
@@ -54,18 +54,15 @@ export const updateDonationPricesAndValues = async (
   currency: string,
   priceChainId: number,
   amount: string | number,
-  chainType: string = ChainType.EVM,
 ) => {
+  logger.info('updateDonationPricesAndValues() has been called', {
+    donationId: donation.id,
+    projectId: project.id,
+    token: token?.symbol,
+    priceChainId,
+  });
   try {
-    if (chainType === ChainType.SOLANA && token) {
```

Comment:
We handle tokens with coingeckoId  below in this part

 else if (token?.coingeckoId) {	    } else if (token?.coingeckoId) {
      const priceUsd = await new CoingeckoPriceAdapter().getTokenPrice({	      const priceUsd = await new CoingeckoPriceAdapter().getTokenPrice({
        symbol: token.coingeckoId,	        symbol: token.coingeckoId,
        networkId: priceChainId,	        networkId: priceChainId,
      });	      });
      donation.priceUsd = toFixNumber(priceUsd, 4);	      donation.priceUsd = toFixNumber(priceUsd, 4);
      donation.valueUsd = toFixNumber(donation.amount * priceUsd, 4);	      donation.valueUsd = toFixNumber(donation.amount * priceUsd, 4);
    }
    
    and it's general, not related to chainType, so I deleted this part and removed chainType from function input params